### PR TITLE
feat: collection update notification type support

### PR DESCRIPTION
**Pull request by DragosIorgulescu**
## Changes

- Added support for the `collection_updated` notification type
- Using the profile picture group to display notification avatars

image

## Events

N / A

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1970 #done

### CODE REVIEW

**Comment by DragosIorgulescu** [comment]
@capJavert I wasn't sure how to generate some collection notifications to test with here, do you have an easy way to do this so I can verify the UI here?

File: packages/shared/src/components/notifications/NotificationItem.tsx [hunk 1]
```diff
@@ -49,17 +50,32 @@ function NotificationItem({
...
-      ?.map?.((avatar) => (
-        <NotificationItemAvatar
-          key={avatar.referenceId}
-          className="z-1"
-          {...avatar}
-        />
-      ))
-      .filter((avatar) => avatar) ?? [];
-  const hasAvatar = avatarComponents.length > 0;
+    type === NotificationType.CollectionUpdated ? (
```
Comment (rebelchris) [inline packages/shared/src/components/notifications/NotificationItem.tsx:56]: Is there anyway we can uniform the two to use the ProfilePictureGroup?
(Think only difference is basically the rounded state.

**Review by rebelchris — CHANGES_REQUESTED** [review]
One request if it's possible? 🙏

File: packages/shared/src/components/notifications/NotificationItem.tsx (hunk 1 above)
Comment (rebelchris) [reply packages/shared/src/components/notifications/NotificationItem.tsx:56]: Hmm I see the spacing is also overlapping for collecitons which is not for the others.

**Review by rebelchris — APPROVED** [review]
Looks good to me on second though :)

File: packages/shared/src/components/notifications/NotificationItem.tsx (hunk 1 above)
Comment (DragosIorgulescu) [reply packages/shared/src/components/notifications/NotificationItem.tsx:56]: good point, this is already modified on the collection post page branch though: https://github.com/dailydotdev/apps/pull/2370/files#diff-1dbb3736e496fd2b9830576074141b705cc949e7eef12d17eb6c0cb5c77211e9

I'll consolidate with those changes here as well

**Comment by capJavert** [comment]
@DragosIorgulescu should be possible to insert some for your user. I can help with that once we merge https://github.com/dailydotdev/daily-api/pull/1558

**Review by sshanzel — APPROVED** [review]
Looks good to me! :shipit:

**Pull request merged**